source/partials/release_notes/_release-19-12-0.md.erb

```diff
@@ -0,0 +1,122 @@
+<h4>Read Only View for Config Repository Pipelines</h4>
+
+Now config repository pipelines' configurations can be viewed within GoCD. Instead of visiting source control repository to know the pipeline settings, GoCD allows these pipelines' configurations to be viewed in read only mode within the application.
```

Comment:
Users will now be able to view configurations for pipelines defined in config repositories in read-only mode within GoCD.